### PR TITLE
[Blazor] Update selenium versions

### PR DESCRIPTION
**Pull request by javiercn**
This PR updates our selenium infrastructure to one that is compatible with the new chrome for testing feature.

### CODE REVIEW

File: src/Components/test/E2ETest/package.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
   "license": "MIT",
   "dependencies": {
     "sauce-connect-launcher": "^1.3.1",
-    "selenium-standalone": "^7.1.0"
+    "selenium-standalone": "^9.1.1"
```
Comment (javiercn) [inline src/Components/test/E2ETest/package.json:16]: Bumped the package to the latest selenium-standalone version

File: src/Shared/E2ETesting/SeleniumStandaloneServer.cs [hunk 2]
```diff
@@ -85,7 +85,7 @@ public static async Task<SeleniumStandaloneServer> GetInstanceAsync(ITestOutputH
     private static async Task InitializeInstance(ITestOutputHelper output)
     {
         var port = FindAvailablePort();
-        var uri = new UriBuilder("http", SeleniumHost, port, "/wd/hub").Uri;
+        var uri = new UriBuilder("http", SeleniumHost, port).Uri;
```
Comment (javiercn) [inline src/Shared/E2ETesting/SeleniumStandaloneServer.cs:88]: This URI no longer exists in Selenium 4.12.0

File: src/Shared/E2ETesting/SeleniumStandaloneServer.cs [hunk 3]
```diff
@@ -111,7 +111,7 @@ private static async Task InitializeInstance(ITestOutputHelper output)
         var psi = new ProcessStartInfo
         {
             FileName = "npm",
-            Arguments = $"run selenium-standalone start -- --config \"{seleniumConfigPath}\" {chromeDriverArg} -- -host {SeleniumHost} -port {port}",
+            Arguments = $"run selenium-standalone start -- --config \"{seleniumConfigPath}\" {chromeDriverArg} -- --host {SeleniumHost} --port {port}",
```
Comment (javiercn) [inline src/Shared/E2ETesting/SeleniumStandaloneServer.cs:114]: Arguments for host and port require double hyphens

File: src/Shared/E2ETesting/selenium-config.json [hunk 4]
```diff
@@ -1,7 +1,8 @@
 {
+  "version": "4.12.0",
```
Comment (javiercn) [inline src/Shared/E2ETesting/selenium-config.json:2]: Uses Selenium 4.12.0, the default version is 4.9.0

**Review by surayya-MS — APPROVED** [review]

File: src/Shared/E2ETesting/SeleniumStandaloneServer.cs (hunk 3 above)
Comment (mkArtakMSFT) [reply src/Shared/E2ETesting/SeleniumStandaloneServer.cs:114]: This is interesting. How this used to work then in the past (or today)?

**Review by mkArtakMSFT — APPROVED** [review]

File: src/Shared/E2ETesting/SeleniumStandaloneServer.cs (hunk 3 above)
Comment (javiercn) [reply src/Shared/E2ETesting/SeleniumStandaloneServer.cs]: It was using an older version

**Comment by SteveSandersonMS** [comment]
Nice

**Comment by javiercn** [comment]
@mkArtakMSFT can you squash and merge this?

File: src/Components/test/E2ETest/package.json [hunk 5]
```diff
@@ -11,11 +11,9 @@
   },
   "author": "",
   "license": "MIT",
-  "dependencies": {
-    "sauce-connect-launcher": "^1.3.1",
-    "selenium-standalone": "^7.1.0"
-  },
   "devDependencies": {
+    "sauce-connect-launcher": "^1.3.1",
+    "selenium-standalone": "^9.1.1",
```
Comment (BrennanConroy) [inline src/Components/test/E2ETest/package.json:16]: Does this mean https://github.com/dotnet/aspnetcore-internal/blob/main/docs/operations/selenium_update.md should be updated?

**Pull request merged**